engine-standalone-storage/src/sync/mod.rs

```diff
@@ -438,6 +438,14 @@ fn non_submit_execute<'db>(
             prev.owner_id = args.clone().new_owner;
             state::set_state(&mut io, &prev)?;
 
+            None
+        }
+        TransactionKind::SetUpgradeDelayBlocks(args) => {
+            let mut prev = state::get_state(&io)?;
+
+            prev.upgrade_delay_blocks = args.clone().upgrade_delay_blocks;
```

Comment:
`clone` is redundant here. You copy primitive type, which implements `Copy` trait by default.
```suggestion
            prev.upgrade_delay_blocks = args.upgrade_delay_blocks;
```